corehq/apps/users/forms.py

```diff
@@ -702,8 +701,7 @@ def __init__(self, project, request_user, *args, **kwargs):
                             strong=_("Good Job! Your password is strong!"),
                             almost=_("Your password is almost strong enough! Try adding numbers or symbols!"),
                             weak=_("Your password is too weak! Try adding numbers or symbols!"),
-                            rules=_("Password Requirements: 1 special character, 1 number, 1 capital letter, "
-                                "minimum length of 8 characters."),
+                            custom_warning=_(settings.CUSTOM_PASSWORD_STRENGTH_MESSAGE),
```

Comment:
is this necessary here? if we're doing a round trip to the server won't the message from the form validation be used anyway?

Reply:
Hey Simon,
This helps in setting the message on the form if you change the password and it does not meet requirements, as a hint message.
Check out working example at
https://icds-staging.commcarehq.org/a/icds-cas/settings/users/commcare/

Reply:
ok